contract/vm.go

```diff
@@ -235,7 +242,7 @@ func (ctx *vmContext) IsGasSystem() bool {
 }
 
 // get the remaining gas from the given LState
-func (ctx *vmContext) getRemainingGas(L *LState) {
+func (ctx *vmContext) refreshRemainingGas(L *LState) {
```

Comment:
this id undoing my changes
there is a pair of functions `getRemainingGas` and `setRemainingGas` that were just put together on #261

Reply:
The previous name looks not close to common naming convention, so I was confused when I firstly saw that function. A function of name starting with `get..` is supposed to return a value and not to change internal field or state, at least in Java. Are there different conventions in golang?

Reply:
OK, I agree. The comment that I added may be sufficient to understand what it is doing